### PR TITLE
Update userDisplayNameField to use "displayName" instead of "gecos"

### PR DESCRIPTION
**Pull request by stephdl**
This pull request updates the `userDisplayNameField` in the `configure-module/20config` file to use the "displayName" field instead of the "gecos" field. 

However 
- when the user has already login, it does not honor the new setting because it has been written to postgres

```
webtop5=# SELECT * FROM core.users;
 domain_id  |   user_id    | type | enabled |               user_uid               |      display_name       |      se
cret      
------------+--------------+------+---------+--------------------------------------+-------------------------+--------
----------
 NethServer | john     | U    | t       | 24e4751b-fc50-4d01-8c65-107a98b7434a | de labrusse             | 3c6sm72
wdpuw22sg
 NethServer | foo       | U    | t       | 3beecb12-63e3-422f-b570-405277fd8cbf | de labrusse             | cqw7q6z
2ydn3i2es
 *          | admin        | U    | t       | 991f72dc-2b96-4340-b88f-53506b160519 | Administrator           | 76gvj7v
bpnuhsf6r
(7 rows)
```

- when you save the configuration after the update of the module, it is not taken except if you change the user domain or the mail server

so in short it is valid only for first installation

https://github.com/NethServer/dev/issues/6923

### CODE REVIEW

**Review by Amygos — APPROVED** [review]

**Comment by Amygos** [comment]
> @Amygos is this fix effective also for already provisioned instances?

No, the fix will apply only on fresh module installations or when the value of `mail_module` and/or`mail_domain` changes, or after a module move/clone.
Also as pointed out by @stephdl the new value will be applied only for the users that log in for the first time.

To avoid the first issue we can resume the work in the PR NethServer/ns8-webtop#19,  so we can always set the correct value in the `configure-module` action or on the module update.

**Comment by stephdl** [comment]
noobs question @Amygos, what is the purpose or the advantage of your solution ?
